packages/orbit-components/src/Box/normalize.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import { WIDTH_AND_HEIGHT, TOKENS } from "./consts";
+import type { ThemeProps, Theme } from "../defaultTheme";
+
+import type { MediaQueryObject, Elevation } from "./index";
+
+const jsxPropsMap = {
+  wrap: "flex-wrap",
+  shrink: "flex-shrink",
+  grow: "flex-grow",
+  direction: "flex-direction",
+  align: "align-items",
+  justify: "justify-content",
+  textAlign: "text-align",
+  maxWidth: "max-width",
+  maxHeight: "max-height",
+};
+
+const isJsxProp = (key): boolean => Object.keys(jsxPropsMap).includes(key);
+
+const formatCSS = (key, value): string => `${key}: ${value};`;
+
+const firstToUpper = string => string.charAt(0).toUpperCase() + string.slice(1);
+
+const normalizeSpacing = (el, property): string[] => {
+  if (typeof el === "object") {
+    return Object.entries(el).map(([key, val]: any) => {
+      return formatCSS(`${property}-${key}`, TOKENS[val]);
```

Comment:
Normalizing spacing like this will block themeability, because the `theme` is directly used and imported in the `consts.js`. It should use the proper value from context provider.

Recommending to put another helper function to this file that would use `theme` property passed to `normalize` function.